packages/adapter-static/README.md

```diff
@@ -38,6 +38,25 @@ The directory to write static assets (the contents of `static`, plus client-side
 
 Specify a fallback page for SPA mode, e.g. `index.html` or `200.html` or `404.html`.
 
+### outputFileName
```

Comment:
I'd expect a new option to go with the pre-rendering options. There's nothing specific about `adapter-static` to it

https://kit.svelte.dev/docs#configuration-prerender